run_behave_tests.py

```diff
@@ -0,0 +1,12 @@
+import os
+import sys
+
+
+if __name__ == '__main__':
+    os.environ['APP_SETTINGS'] = 'TestConfig'
+
+    from behave import __main__ as behave_executable
+    behave_errors = behave_executable.main(args="--format progress")
```

Comment:
would not be good to loop over the rrors and print them in a human redable way so that we can aknowledge the issues ?